### DISCUSSION TITLE
Working While Minimize

### DISCUSSION THREAD

**Discussion by Dewruwan95** [rizwansoaib/whatsapp-monitor#174] [General]
Both Windows app and extension work me properly, but they didn't work after minimizing the app or browser window. is there any method to work while minimizing?

**Comment by rizwansoaib** [comment]
if you don't do any activity for more than 2min and screen is opened is it also working?

Pleas check again WhatsApp updated no matter you open window or not you need to do some activity like moving mouse or click somewhere in WhatsApp window then it will able to track.

**Reply by Dewruwan95** [reply]
it works, but if I minimize the window or move to another desktop in windows multi-desktop and after more than 5 minutes it stops working. is it must I be online always?

**Comment by rizwansoaib** [comment]
Yes that's what I am saying for minimising window then see online activity
is currently not working but if you opened screen for moving mouse use
pyautogui
Please refer #145

On Sat, 21 May, 2022, 9:53 am Ositha Dewruwan, ***@***.***>
wrote:

> it works, but if I minimize the window or move to another desktop in
> windows multi-desktop and after more than 5 minutes it stops working. is it
> must I be online always?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/174#discussioncomment-2794678>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZDNJPHVFTFRTN67VI3VLBQL5ANCNFSM5WRE3JCQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> .com>
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Reply by Dewruwan95** [reply]
Ok. Understood. Thanks 💪
